dangerzone/cli.py

```diff
@@ -19,76 +18,48 @@ def print_header(s: str) -> None:
     click.echo(Style.BRIGHT + s)
 
 
-@click.command()
-@click.option("--output-filename", help="Default is filename ending with -safe.pdf")
+@click.command(context_settings={"allow_interspersed_args": False})
```

Comment:
We should handle interspersed args the same way git does: if it detects potential args on the files list, it warns the user:
```
$ git log dangerzone/gui/systray.py -p
fatal: option '-p' must come before non-option arguments
```